src/components/PetitionForm/fields.js

```diff
@@ -52,6 +52,19 @@ export default [
     }
   },
   {
+    element: 'InputRepeater',
+    name: 'links[]',
+    label: settings.petitionFields.links.label,
+    hint: settings.petitionFields.links.hint,
+    addInputLabel: settings.petitionFields.links.addInputLabel,
+    removeInputLabel: settings.petitionFields.links.removeInputLabel,
+    maxInputCount: 3,
+    html: {
+      type: 'url',
```

Comment:
The default HTML5 URL validation actually sucks. I could maybe do something better with patterns, or just let Javascript handle it and set it as a text input.